R/add_labels.R

```diff
@@ -0,0 +1,157 @@
+#' @title Add variable and value labels
+#' @name add_labels
```

Comment:
I know you wrote it but might be good to avoid name conflicts with `sjlabelled::add_labels()`?

Reply:
haha, I was only thinking of `set_label()` and `set_labels()`. Damn... Any suggestions for a new name?

Reply:
`append_label`? `put_label()`? I'm not convinced by either

Reply:
also `assign_label()`, `apply_label()`, `give_label()`?

Reply:
chat GPT suggests `map_labels()`.

Reply:
or something like `data_labels()` or similar...?

Reply:
I think we keep `data_` functions for dataframes only (and not a big fan of `map_labels`)

Reply:
or we don't care if we add another name conflict? There are already some with _sjlabelled_ and _sjmisc_. The long term goal would be to soft-deprecate sjverse and switch to easystats, anyway ;-)

Reply:
or just `add_lab()` and `add_labs()` ?

Reply:
yeah but there's `ggplot2::labs()` so it doesn't come out of nowhere ^^

Reply:
> or just add_lab() and add_labs()

I would prefer to have just one function that sets value and/or variable labels depending on the provided arguments. So it would be `add_labs()`.

Reply:
Of the mentioned suggestions, I like `set_labels()` the best.

Reply:
That would also conflict with _sjlabelled_.

Reply:
Damn.
How different are these functions from _sjlabelled_? 
Is there a possibility to re-export the _datawizard_ versions of these functions from _sjlabelled_?